generate-api-reference.js

```diff
@@ -0,0 +1,58 @@
+const fs = require('fs');
+const yaml = require('js-yaml');
+
+const API_SCHEMA_URL =
+  'https://raw.githubusercontent.com/suhailkakar/livepeer-studio/master/packages/api/src/schema/api-schema.yaml';
```

Comment:
Will we need to update this with the production repo?

Reply:
Yes, I am waiting for this pr to get merged: https://github.com/livepeer/studio/pull/1754